Libplanet.Tests/Net/SwarmTest.Preload.cs

```diff
@@ -1167,5 +1167,97 @@ public async Task PreloadAsyncCancellation(int cancelAfter)
                 );
             }
         }
+
+        [Fact(Timeout = Timeout)]
+        public async void GetDemandBlockHashes()
```

Comment:
`async void` isn't traceable since it doesn't return `Task`.

```suggestion
        public async Task GetDemandBlockHashes()
```

Reply:
Fixed in 25e887051f8b20fb9f72c80bf00dc11a319d95aa (amended).